src/views/Export/ExportHeader/index.js

```diff
@@ -48,6 +45,22 @@ const createReportStructureForExport = nodes => nodes
         }
     ));
 
+const createReportStructureLevelForExport = (nodes) => {
+    const a = nodes
+        .filter(node => node.selected)
+        .map(node => (
+            node.nodes ? {
+                id: node.key,
+                title: node.title,
+                sublevels: createReportStructureLevelForExport(node.nodes),
+            } : {
+                id: node.key,
+                title: node.title,
+            }
```

Comment:
Why not just do:
`sublevels: node.nodes ? createReportStructureLevelForExport(node.nodes) : undefined`